src/stim/stabilizers/tableau.pybind.cc

```diff
@@ -2130,8 +2130,16 @@ void stim_pybind::pybind_tableau_methods(pybind11::module &m, pybind11::class_<T
             }
 
             std::vector<std::complex<float>> v;
+            double weight = 0;
```

Comment:
This should be moved into the actual C++ method being called, instead of only affecting the python API.

Reply:
I moved it to the underlying C++ method, but after writing the test, I realized normalization already happens within VectorSimulator: https://github.com/quantumlib/Stim/blob/49531a44bb1c412a0186b171e1c0076833707059/src/stim/simulators/vector_simulator.h#L79. So, we don't **need** to explicitly need to pre-normalize the input, or check for zero, but let me know if you'd prefer it that way.